AUTHORS

```diff
@@ -744,7 +744,7 @@ JMSS-Unknown <31131631+JMSS-Unknown@users.noreply.github.com>
 Arshdeep Singh <singh.arshdeep1999@gmail.com>
 cym1 <16437732+cym1@users.noreply.github.com>
 Stewart Wadsworth <stewart.wadsworth@gmail.com>
-Jared Lumpe <mjlumpe@gmail.com>
+Michael Jared Lumpe <mjlumpe@gmail.com>
```

Comment:
@jlumpe do you prefer "Michael Jared Lumpe" or "Jared Lumpe" in the AUTHORS file?

Reply:
I think I'm trying to keep everything as just "Jared Lumpe" everywhere. Thanks!

Reply:
Fixed.